.circleci/config.yml

```diff
@@ -30,7 +30,7 @@ jobs:
           pkg-manager: pip
       - run:
           name: Test adapter on SQL Server against dbt-adapter-tests
-          command: tox -e integration-sqlserver
+          command: tox -- -v test/integration/sqlserver.dbtspec
```

Comment:
this is how `tox` recommends you to [pass down positional arguments](https://tox.readthedocs.io/en/latest/example/general.html#interactively-passing-positional-arguments) together with {posargs}.